docs/sites.yml

```diff
@@ -2078,3 +2078,14 @@
   built_by: Mill3 Studio
   built_by_url: https://mill3.studio/en/
   featured: false
+  - title: Bestvideogame.deals
+  main_url: https://bestvideogame.deals/
+  url: https://bestvideogame.deals/
+  description: >
+    Video game comparison website for the UK, build with GatsbyJS.
+  categories:
+    - Price comparison
+    - Video gaming
```

Comment:
I'd probably change the categories to eCommerce and Video Games, but that's subjective so I'll leave that up to you. See https://github.com/gatsbyjs/gatsby/blob/master/docs/docs/site-showcase-submissions.md#categories for more information on that.

Reply:
Changed! Thanks @wKovacs64